plugins/UserCountryMap/Controller.php

```diff
@@ -286,10 +287,7 @@ private function getMetrics($idSite, $period, $date, $token_auth)
         $metrics = array();
         if (!empty($metaData[0]['metrics']) && is_array($metaData[0]['metrics'])) {
             foreach ($metaData[0]['metrics'] as $id => $val) {
-                // todo: should use SettingsPiwik::isUniqueVisitorsEnabled ?
-                if (Common::getRequestVar('period') == 'day' || $id != 'nb_uniq_visitors') {
-                    $metrics[] = array($id, $val);
-                }
+                $metrics[] = array($id, $val);
```

Comment:
That check should not be needed, as the `API.getMetadata` should only return metrics that are available for the given period.